packages/api/src/endpoints/eventSub/HelixEventSubApi.ts

```diff
@@ -1189,6 +1189,26 @@ export class HelixEventSubApi extends BaseApi {
 		);
 	}
 
+	/**
+	 * Subscribe to events that represent an Ad Break beginning in a channel.
```

Comment:
Not sure why "Ad Break" would be capitalized here (nor in any EventSub classes).